api-compact.yaml

```diff
@@ -60,22 +60,22 @@ auth_header: &auth_header
   type: "string"
 
 limit_qs_parameter: &limit_qs_parameter
-  in: query
-  name: limit
+  in: "query"
+  name: "limit"
```

Comment:
Nevermind, I didn't realize there was already a convention which the existing code was straying from.